### ISSUE TITLE
Android issue - FileProvider

### ISSUE THREAD

**Issue by maddentist** [jamesmontemagno/MediaPlugin#861]
## Bug Information

Version Number of Plugin: 5.0.1
Device Tested On: Google Pixel 3XL
Version of VS: 16.5.5
Version of Xamarin: 4.6.0.772

### Steps to reproduce the Behavior
Create a very simple xamarin.forms app with a button to take picture with CrossMedia.Current.TakePhotoAsync().

Steps has been followed to configure Android with this in android Manifest
`<provider
     android:name="androidx.core.content.FileProvider"
     android:authorities="${applicationId}.fileprovider"
     android:exported="false"
     android:grantUriPermissions="true">
      <meta-data
          android:name="android.support.FILE_PROVIDER_PATHS"
          android:resource="@xml/file_paths" />
    </provider>`

and

 `<?xml version="1.0" encoding="utf-8"?>
<manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="1" android:versionName="1.0" package="com.companyname.tattox" android:installLocation="internalOnly">
	<uses-sdk android:minSdkVersion="24" android:targetSdkVersion="29" />
	<application android:label="TattoX.Android">
		<activity android:name="microsoft.identity.client.BrowserTabActivity">
			<intent-filter>
				<action android:name="android.intent.action.VIEW" />
				<category android:name="android.intent.category.DEFAULT" />
				<category android:name="android.intent.category.BROWSABLE" />
				<data android:scheme="msauth" android:host="com.companyname.tattox" />
			</intent-filter>
		</activity>
   
     <provider
     android:name="androidx.core.content.FileProvider"
     android:authorities="${applicationId}.fileprovider"
     android:exported="false"
     android:grantUriPermissions="true">
      <meta-data
          android:name="android.support.FILE_PROVIDER_PATHS"
          android:resource="@xml/file_paths" />
    </provider>
	</application>
	<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
	<uses-permission android:name="android.permission.INTERNET" />
	<uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
	<uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
</manifest>`

for file_path.xml.

Basically, copy paste from the website.

### Expected Behavior
The app should be able to compile and launch.

### Actual Behavior
The app compiles and throws with the following: 
The android app will not launch displaying the following exception:
**Java.Lang.RuntimeException
  Message=Unable to get provider androidx.core.content.FileProvider: java.lang.ClassNotFoundException: Didn't find class "androidx.core.content.FileProvider" on path: DexPathList[[zip file "/data/app/com.companyname.tattox-Mu05AzlR3YsYJpS6Cl_k8A==/base.apk"],nativeLibraryDirectories=[/data/app/com.companyname.tattox-Mu05AzlR3YsYJpS6Cl_k8A==/lib/arm64, /data/app/com.companyname.tattox-Mu05AzlR3YsYJpS6Cl_k8A==/base.apk!/lib/arm64-v8a, /system/lib64, /product/lib64]]**

**Comment by jamesmontemagno** [comment]
are you using androidx?

**Comment by maddentist** [comment]
> are you using androidx?

I do think so, do you have an efficient way of being sure (I think that newly created xamarin prjects for Android use androidx now. Furthermore, no legacy androidsupport lib have been added to the project by vs)

**Comment by jamesmontemagno** [comment]
You would have to be compiling Against Android 10 and updating to the latest nugets. 

You would just see AndroidX stuff in the nugets. Sounds like you are.

Might just need to update to latest nugets or turn off multi-dex. Looks like an Android issue: https://stackoverflow.com/questions/52696574/didnt-find-class-androidx-core-content-fileprovider

**Comment by ashishjangir** [comment]
<application android:label="Demoapp.Android" android:usesCleartextTraffic="true" android:requestLegacyExternalStorage="true">
		<provider android:name="androidx.core.content.FileProvider" android:authorities="${applicationId}.provider" android:exported="false" android:grantUriPermissions="true">
			<meta-data android:name="android.support.FILE_PROVIDER_PATHS" android:resource="@xml/provider_paths" />
		</provider>
	</application>

**Comment by rdileep** [comment]
Updating xamarin.forms to 5.0.0.2083 got fixed.
